web/src/app/components/dashboard/dashboard.component.ts

```diff
@@ -45,7 +48,11 @@ export class DashboardComponent implements OnInit {
     this.userService.getUserDashboardDetails(provider, displayName)
       .subscribe((value) => {
         this.getUserSocialDetails(value.uid);
```

Comment:
This is still here in the subscribe and will be async without any wait, it should go in a pipe before the subscribe